.github/workflows/release-manual.yml

```
@@ -0,0 +1,52 @@
+        uses: ./.github/actions/setup-node
+        with:
+          node-version: 22.16.0
+          deno-version: 1.43.5
+          cache-modules: true
+          install: true
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - uses: rharkor/caching-for-turbo@v1.8
+
+      - name: 'Start release: ${{ inputs.action }}'
+        uses: ./packages/release-action
```

actionlint (1.7.11)

[error] file "dist/index.js" does not exist in "/home/jailuser/git/packages/release-action". it is specified at "main" key in "runs" section in "Changeset release" action

(action)